CMarkGFM.hsc

```diff
@@ -73,7 +72,7 @@ extsToLlist (h:t) = do
 
 -- | Resolves a CMarkExtension to its pointer.
 resolveExt :: CMarkExtension -> IO (Maybe ExtensionPtr)
-resolveExt e = do
+resolveExt e = registerPlugins `seq` do
```

Comment:
I think that’s true; `pseq` is more of a parallelism primitive. That said, I think it might be more idiomatic to leave `registerPlugins` in `IO` and add the action to the `do` block before the rest of the operations:

```haskell
resolveExt e = do
  registerPlugins
  p <- withCString …
  …
```

If we wanted to ensure that we only call it once, then it seems like the right place to enforce that invariant would be inside `registerPlugins` itself, or even in the C code using a global flag. Not sure how important that is to the API, tho.